### PR TITLE
MAT-6788 fix execution error after measure retransferred

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-6788](https://jira.cms.gov/browse/MAT-6788)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java [hunk 1]
```diff
@@ -32,7 +32,35 @@ public Measure overwriteExistingMeasure(
...
+                              .createdAt(testCase.getCreatedAt())
+                              .createdBy(testCase.getCreatedBy())
+                              .lastModifiedAt(testCase.getLastModifiedAt())
+                              .lastModifiedBy(testCase.getLastModifiedBy())
+                              .resourceUri(testCase.getResourceUri())
+                              .validResource(testCase.isValidResource())
+                              .json(testCase.getJson())
+                              .patientId(testCase.getPatientId())
+                              .hapiOperationOutcome(testCase.getHapiOperationOutcome())
+                              .build())
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java]: maybe we could just create a new copy and reset groups? e.g. `testCase.toBuilder().groupPopulations.(null/List.of()).build()`

Comment (jkotanchik-SB) [inline src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java]: Lombok provides a shortcut for duplicating objects:
```suggestion
                    testCase -> testCase.toBuilder().build())
```

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java]: @adongare thanks will do.

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java]: @jkotanchik-SB thanks so much will do.

**Review by adongare — APPROVED** [review]

File: src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java (hunk 1 above)
Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java]: @jkotanchik-SB I did the change can I resolve this conversation? Thanks
